### ISSUE TITLE
The comfort vignette is broken

### ISSUE THREAD

**Issue by pepa-husek** [DrBeef/JKXR#20]
The comfort vignette on Quest seems to be broken and displays in the middle of the FOV, instead of at the edges. I think perhaps the referenced line and the following one (with the hardcoded values), might be to blame? 
https://github.com/DrBeef/JKXR/blob/69776766561fc0619e6cce7bf466488ebca4aad3/Projects/Android/jni/OpenJK/codeJK2/cgame/cg_draw.cpp#L2655

**Comment by geomusicmaker** [comment]
also experiencing this. A working comfort vignette plus the ability to slow movement speed would go a long way to help motion sickness.

**Comment by pepa-husek** [comment]
exactly, it's a great port and I'm sure adding an implementation of teleport movement into the existing game would be quite difficult... but the vignette + slow walking would still help a lot. This way I'm only able to play like 10-15 minutes if I don't want to be totally nauseous :/
